azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst

```diff
@@ -0,0 +1,137 @@
+Provision an Ubuntu virtual machine running SQL Server in Azure
+===============================================================
+
+This documentation is based on the `Ubuntu tutorial <https://ubuntu.com/tutorials/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure#2-launch-an-ubuntu-vm-with-sql-server-installed>`_
+for creating Linux virtual machines with the Azure CLI.
+
+
+Prerequisites
+-------------
+
+- A Microsoft Azure account
+
+
+Launch an Ubuntu VM with SQL Server installed
+---------------------------------------------
+
+
+Find the correct image to launch on the Azure portal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Sign in to the Azure portal. https://portal.azure.com/
+
+- In the left pane, select Create a resource.
+
+- In the search box, type “SQL 2019 on Ubuntu Pro”, and submit the search.
+
+- Select thew image by the name “SQL 2019 on Ubuntu Pro 20.04 LTS with 24x7 Support”.
```

Comment:
Out of curiosity, why is Focal being used in this example?

Reply:
This is the example from the original tutorial. I've left examples exactly as they appeared there.